src/include/checkpoint.h

```diff
@@ -9,24 +9,22 @@
 #define WT_CHECKPOINT_SESSION_FLAGS (WT_SESSION_CAN_WAIT | WT_SESSION_IGNORE_CACHE_SIZE)
 
 #define WT_CHECKPOINT_STATE_INACTIVE 0
-#define WT_CHECKPOINT_STATE_RESET_CURSORS 1
```

Comment:
I no longer think we need to remove these. If nothing is running besides a proper checkpoint, then we don't have the concurrency issues that made us remove these in the first place.

Reply:
We still cannot set anything outside of the checkpoint lock because if you queue a checkpoint, it will overwrite the state.